tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -3028,16 +3055,30 @@ int BPF_KPROBE(trace_security_socket_bind)
     if (connect_id.port) {
         net_ctx_t net_ctx;
         net_ctx.host_tid = context.host_tid;
+        __builtin_memcpy(net_ctx.comm, context.comm, TASK_COMM_LEN);
         bpf_map_update_elem(&network_map, &connect_id, &net_ctx, BPF_ANY);
     }
 
     events_perf_submit(ctx);
 
+    // netDebug event
+    if (get_config(CONFIG_DEBUG_NET)) {
+        net_debug_t debug_event = {0};
+        debug_event.ts = bpf_ktime_get_ns()/1000;
```

Comment:
1199583.960716  nc                2618160  udp_sendmsg          LocalIP: 127.0.0.1, LocalPort: 36895, Protocol: 17
1199583.960755  nc                2618160  packet               Len: 47, SrcIP: 127.0.0.1, SrcPort: 36895, DestIP: 127.0.0.1, DestPort: 14219, Protocol: 17
1199583.960771  nc                2618160  packet               Len: 47, SrcIP: 127.0.0.1, SrcPort: 36895, DestIP: 127.0.0.1, DestPort: 14219, Protocol: 17

Timestamp from debug-net looks good. I think only the pcap timestamp generation was broken (like said in the other commit's comment).

Reply:
BTW as you can see I went from commit to commit (That is why debug port is still broken in this output). I hadn't applied the little->big endian fix yet.